lib/build.js

```diff
@@ -60,6 +60,7 @@ module.exports = {
     this.stdout = new Buffer(0);
     this.stderr = new Buffer(0);
     atom.commands.add('atom-workspace', 'build:trigger', this.build.bind(this));
+    atom.commands.add('atom-workspace', 'build:toggle-view', this.toggleView.bind(this));
```

Comment:
Never mind, it seems that registering the same command multiple times is not an issue according to the docs.